cardano-cli/src/Cardano/CLI/Shelley/Run/Query.hs

```diff
@@ -796,6 +802,11 @@ obtainLedgerEraClassConstraints
 obtainLedgerEraClassConstraints ShelleyBasedEraShelley f = f
 obtainLedgerEraClassConstraints ShelleyBasedEraAllegra f = f
 obtainLedgerEraClassConstraints ShelleyBasedEraMary    f = f
-obtainLedgerEraClassConstraints ShelleyBasedEraAlonzo  _ =
-  panic "obtainLedgerEraClassConstraints: Alonzo era not implemented yet"
-
+obtainLedgerEraClassConstraints ShelleyBasedEraAlonzo  f = f
+
+instance ToJSON (Alonzo.TxOut (Alonzo.AlonzoEra StandardCrypto)) where
+  toJSON (Alonzo.TxOut addr value dataHash) = Aeson.object
+    [ "addr" .= toJSON addr
+    , "value" .= toJSON value
+    , "dataHash" .= toJSON dataHash
```

Comment:
`TxOut` field names.

Reply:
Sorry, what is that comment about?

Reply:
I added to comment so I can make a permalink to share with the ledger team.

Reply:
We already have this instance in `Cardano.Tracing.OrphanInstances.Shelley`. What we should do is move the instance in `Cardano.Tracing.OrphanInstances.Shelley` to `Cardano.Api.Orphans` and import it instead.